plan/optimizer.go

```diff
@@ -27,6 +27,9 @@ import (
 // AllowCartesianProduct means whether tidb allows cartesian join without equal conditions.
 var AllowCartesianProduct = true
 
+// EnableStatistic means tidb will use the statistic information to do cost-based information.
```

Comment:
`EnableStatistic means whether tidb uses the statistic information to do cost-based optimization` is more uniform with `AllowCartesianProduct`.